src/classes/TDTM_TriggerHandler.cls

```diff
@@ -197,20 +196,7 @@ public class TDTM_TriggerHandler {
     * @param dmlWrapper The class that contains the records that need to have DML performed on.
     * @return void
     */
-    public static void processDML(TDTM_Runnable.DmlWrapper dmlWrapper) {
-        processDMLWithRollback(dmlWrapper, true);
-    }
-    /*******************************************************************************************************
-    * @description Performs pending DML operations, on the records stored in the DMLWrapper that it gets as parameter.
-    * We want this method to be static so that it can be called from the future method in TDTM_Runnable.
-    * @param dmlWrapper The class that contains the records that need to have DML performed on.
-    * @param withRollback True if errors should cause rollback, false if not.
-    * @return Errors Object with list of Error objects
-    */
-    public static ERR_Handler.Errors processDMLWithRollback(TDTM_Runnable.DmlWrapper dmlWrapper, boolean withRollback) {
```

Comment:
This method is being removed. It was originally created in NPSP just so the Household Addresses code could call it, with the second parameter set to false. Now that code is doing its own DML and error processing.